### PR TITLE
[DOCS-6812] Update MySQL database and driver versions for ACS 7.2.1

### PR DESCRIPTION
**Pull request by anxumalo**

### CODE REVIEW

File: content-services/latest/support/index.md
Comment (anxumalo) [reply content-services/latest/support/index.md:25]: Good spot! I've only updated the content for ACS 7.2.1, and not back-ported to ACS 7.2.0 - waiting on a review from @damianujma to confirm this is the right thing to do.

Comment (damianujma) [reply content-services/latest/support/index.md:25]: LGTM. I did the change only for ACS **7.2.1.** It will be better to ask @tiagosalvado10 because he worked on this issue [ACS-3271](https://alfresco.atlassian.net/browse/ACS-3271) for other ACS versions.

**Review by damianujma — APPROVED** [review]

File: content-services/latest/support/index.md
Comment (anxumalo) [reply content-services/latest/support/index.md:25]: I spoke to Tiago - no further changes needed in this PR so I'll resolve this conversation & merge the update.

Comment (anxumalo) [reply content-services/latest/support/index.md:25]: Thanks for the reviews @Amy-Hyland & @damianujma

**Pull request merged**
